lnbits/core/models.py

```diff
@@ -79,14 +79,25 @@ class WalletTypeInfo:
     wallet: Wallet
 
 
+class UserConfig(BaseModel):
+    email_verified: Optional[bool] = False
+    first_name: Optional[str] = None
+    last_name: Optional[str] = None
+    display_name: Optional[str] = None
+    picture: Optional[str] = None
+    provider: Optional[str] = "lnbits"  # auth provider
```

Comment:
is there a reason why you added this into a separate model/object `config:` . i would rather have it flat in the user table/model so you could do a simple `SELECT * from users WHERE email_verified=true`. the less json in the db the better, imho

Reply:
> i would rather have it flat in the user table/model
- if there is a scenario where one of this fields is required for querying then it can be extracted. At the moment there is none so we should not over-generalize

> the less json in the db the better, imho
- if the data is not needed for querying then it is fine in a JSON, no need to have data migrations each time a new field is added to the model